src/datarobotx/idp/llm_blueprints.py

```diff
@@ -103,16 +113,38 @@ def get_or_register_llm_blueprint_custom_model_version(
         The DataRobot API token to use.
     llm_blueprint_id : str
         The ID of the LLM blueprint to use.
+    guard_kwargs : Optional[List[Dict[str, Any]]
+        Additional arguments to configure LLM guards.
```

Comment:
Think this should be made a bit more clear - are these literally the kwargs that will be handed to method X in file Y? e.g. can a user go read the docstring for method X to learn more about what valid kwargs are besides the below example?